[2024-03-14T23:49:54.025] [INFO] CHEESE - /get/openid: [params] { a: '1' }
[2024-03-14T23:49:59.416] [INFO] CHEESE - /get/openid: [result] {
  errcode: 41008,
  errmsg: 'missing code, rid: 65f31ca8-6a2a8e32-65edf177'
}
[2024-03-14T23:50:02.978] [INFO] CHEESE - error connecting: Error: Cannot enqueue Handshake after fatal error.
    at Protocol._validateEnqueue (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Protocol.handshake (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/Connection.js:116:18)
    at queryDb (/Users/ddj/Documents/mini-server/src/dbHandler.js:16:14)
    at Layer.handle [as handle_request] (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/ddj/Documents/mini-server/node_modules/express/lib/router/index.js:284:15
[2024-03-14T23:50:02.979] [INFO] CHEESE - error connecting: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/Users/ddj/Documents/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at queryDb (/Users/ddj/Documents/mini-server/src/dbHandler.js:25:14)
    at Layer.handle [as handle_request] (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/ddj/Documents/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/ddj/Documents/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:16:39.416] [INFO] CHEESE - error connecting: Error: ER_BAD_DB_ERROR: Unknown database 'sport_record'
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:116:18)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:16:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:16:39.420] [INFO] CHEESE - error connecting: Error: ER_BAD_DB_ERROR: Unknown database 'sport_record'
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:116:18)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:16:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:16:48.869] [INFO] CHEESE - error connecting: Error: Cannot enqueue Handshake after fatal error.
    at Protocol._validateEnqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Protocol.handshake (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:116:18)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:16:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
[2024-04-07T20:16:48.870] [INFO] CHEESE - error connecting: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:25:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:21:01.326] [INFO] CHEESE - error connecting: Error: Cannot enqueue Handshake after fatal error.
    at Protocol._validateEnqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Protocol.handshake (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:116:18)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:16:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
[2024-04-07T20:21:01.328] [INFO] CHEESE - error connecting: Error: Cannot enqueue Query after fatal error.
    at Protocol._validateEnqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:212:16)
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at queryDb (/Users/jdd/person/mini-server/src/dbHandler.js:25:14)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:21:13.288] [INFO] CHEESE - connected as id 15
[2024-04-07T20:21:13.301] [INFO] CHEESE - The solution is:  []
[2024-04-07T20:24:52.547] [INFO] CHEESE - error connecting: Error: ER_NO_SUCH_TABLE: Table 'sport_record.sport_record.record' doesn't exist
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Object.insertRecord (/Users/jdd/person/mini-server/src/dbHandler.js:41:14)
    at /Users/jdd/person/mini-server/src/index.js:12:13
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:27:14.154] [INFO] CHEESE - error connecting: Error: ER_NO_SUCH_TABLE: Table 'sport_record.sport_record.record' doesn't exist
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Object.insertRecord (/Users/jdd/person/mini-server/src/dbHandler.js:41:14)
    at /Users/jdd/person/mini-server/src/index.js:12:13
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:27:17.223] [INFO] CHEESE - error connecting: Error: ER_NO_SUCH_TABLE: Table 'sport_record.sport_record.record' doesn't exist
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Object.insertRecord (/Users/jdd/person/mini-server/src/dbHandler.js:41:14)
    at /Users/jdd/person/mini-server/src/index.js:12:13
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:27:49.806] [INFO] CHEESE - error connecting2: Error: ER_NO_SUCH_TABLE: Table 'sport_record.sport_record.record' doesn't exist
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Object.insertRecord (/Users/jdd/person/mini-server/src/dbHandler.js:41:14)
    at /Users/jdd/person/mini-server/src/index.js:12:13
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:28:10.728] [INFO] CHEESE - error connecting2: Error: ER_NO_SUCH_TABLE: Table 'sport_record.sport_record.record' doesn't exist
    at Sequence._packetToError (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Object.insertRecord (/Users/jdd/person/mini-server/src/dbHandler.js:41:14)
    at /Users/jdd/person/mini-server/src/index.js:12:13
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/jdd/person/mini-server/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/Users/jdd/person/mini-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/jdd/person/mini-server/node_modules/express/lib/router/index.js:346:12)
[2024-04-07T20:28:44.024] [INFO] CHEESE - error connecting1: Error: Cannot enqueue Query after invoking quit.
    at Protocol._validateEnqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:215:16)
    at Protocol._enqueue (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:138:13)
    at Connection.query (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:198:25)
    at Query.<anonymous> (/Users/jdd/person/mini-server/src/dbHandler.js:60:18)
    at Query.<anonymous> (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:526:10)
    at Query._callback (/Users/jdd/person/mini-server/node_modules/mysql/lib/Connection.js:488:16)
    at Sequence.end (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
    at Query._handleFinalResultPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:149:8)
    at Query.EofPacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/sequences/Query.js:133:8)
    at Protocol._parsePacket (/Users/jdd/person/mini-server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
[2024-04-07T20:30:42.595] [INFO] CHEESE - connected as id 25
[2024-04-07T20:30:42.601] [INFO] CHEESE - The solution is:  [ RowDataPacket { openid: '1', count: 1 } ]
